### DISCUSSION TITLE
Creality CR-10 Max settings.

### DISCUSSION THREAD

**Discussion by IHateNameStealers** [supermerill/SuperSlicer#1799] [Q&A] [answered]
My dad gave me a CR-10 Max to start playing with.
The slicer that came with it was the Creality Slicer.
It does it's job, but I would love to start using SuperSlicer with my CR-10 Max.

I have copied most of the settings from the Creality Slicer into SuperSlicer, but the 3D prints made with SS are now way as good looking as the ones covnerted by Creality Slicer.

Anyone here who would like to share his settings for the CR-10 Max?
I'm using a 0.4 nozzle and 1.75mm filament

Thank you

**Comment by supermerill** [comment, marked as answer]
There is the cr-10 max in the wizard (in the cr family). if you find some problems with the profile, tell me what needs to change.

**Reply by IHateNameStealers** [reply]
Hmmm I must have missed than one. 
Thank you for the info.
